tests/CMakeLists.txt

```diff
@@ -473,6 +473,10 @@ set_tests_properties("Reporters::DashAsLocationInReporterSpecSendsOutputToStdout
 )
 
 if (CATCH_ENABLE_CONFIGURE_TESTS)
+    add_test(NAME "CMakeConfig::ConfigureBazelReporter"
+      COMMAND
+        "${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_LIST_DIR}/TestScripts/testConfigureBazelReporter.py" "${CATCH_DIR}" "${CMAKE_CURRENT_BINARY_DIR}"
+    )
```

Comment:
I don't think this is worth a full configure tests -- they are _very_ expensive.

Instead, I would add another separately-compiled test into `tests/ExtraTests/CMakeLists.txt`. See tests linking into `Catch2_buildall_interface` on how to do that.

Reply:
I went this route because the test needs to set the `XML_OUTPUT_FILE` var and check that the file is correct as opposed to a simple regex match. 
I can try to refactor this into a seperately compiled test which is wrapped in a python script to do this 👍

Reply:
I refactored the test as you suggested, wasn't sure about the prefix of the new file so let me know what you think 👍